src/js/providers/html5.ts

```diff
@@ -809,6 +810,14 @@ function VideoProvider(this: HTML5Provider, _playerId: string, _playerConfig: Ge
         if (!_canSeek) {
             _canSeek = !!_getSeekableEnd();
         }
+
+        // Workaround for a bug in iOS where you are unable to seek in initially muted streams
+        if (OS.iOS && _videotag.muted && !iosMuteToggle) {
+            _videotag.muted = false;
+            _videotag.muted = true;
+        }
+        iosMuteToggle = true;
```

Comment:
`iosMuteToggle = true` can go on the line folling `if (`!iosMuteToggle) {`.

You can also flip it so it starts `true` but only when we need to perform it:

`let iosMuteToggle =  OS.iOS || Borwser.Safari;`

then `if (iosMuteToggle && _videotag.muted) {`

but keep in mind we might need to do this for every playlist item that starts muted in Safari so you need to reset `iosMuteToggle` on load or maybe even canplay (careful we found that canplay can fire multiple times for a video in Safari now).